fenix/app/src/main/res/values/strings.xml

```diff
@@ -393,21 +394,27 @@
     <!-- Text for request support button indicating that cookie banner reduction is not supported for the current site, this is shown as part of the cookie banner details view. -->
     <string name="cookie_banner_handling_details_site_is_not_supported_request_support_button">Request support</string>
     <!-- Text for title indicating that cookie banner reduction is not supported for the current site, this is shown as part of the cookie banner details view. -->
-    <string name="cookie_banner_handling_details_site_is_not_supported_title">Cookie Banner Reduction</string>
+    <string name="cookie_banner_handling_details_site_is_not_supported_title" moz:RemovedIn="114" tools:ignore="UnusedResources">Cookie Banner Reduction</string>
+    <string name="cookie_banner_handling_details_site_is_not_supported_title_2">Request support for this site?</string>
```

Comment:
I think we need to duplicate the translator's comment it could cause confusion, as they will see the new string but not the comment description.

Reply:
Done